docs/source/gettingstarted.rst

```diff
@@ -39,6 +39,11 @@ is to enable the NeuroDebian_ repository, and to install datalad as a system
 package::
 
   sudo apt-get install datalad
+  
+A current version of git-annex (as also provided by the NeuroDebian_ 
+repository) can be installed by typing::
+
+  sudo apt-get install git-annex
```

Comment:
actually hold on -- whenever you 'apt-get install datalad' -- it should install needed version of annex automagically... if not -- then it is a bug in packaging which we should fix but I think it should be alright, isn't it?